public/js/app/components/Table.jsx

```diff
@@ -1,28 +1,33 @@
 import React from 'react'
 
-const renderRow = (row) => (
-  <tr key={row[0]}>
-    {row.map((cell, index) => (
-      <td key={index}>{typeof cell === 'function' ? cell() : cell}</td>
-    ))}
-  </tr>
-)
+const renderRow = (row, rowIndex) => {
+  return (
+    //TODO: unique ID for key
+    <tr key={rowIndex}>
+      {row.map((cell, cellIndex) => (
+        <td key={cellIndex}>{typeof cell === 'function' ? cell() : cell}</td>
```

Comment:
Do we actually have a function as a cell somewhere in our code? If yes, can we find out whether we can call that function already when we create the data as opposed to when we display it?